src/main/java/uk/gov/hmcts/reform/blobrouter/exceptions/EnvelopeNotFoundException.java

```diff
@@ -1,4 +1,10 @@
 package uk.gov.hmcts.reform.blobrouter.exceptions;
 
 public class EnvelopeNotFoundException extends RuntimeException {
+    public EnvelopeNotFoundException() {
+    }
+
+    public EnvelopeNotFoundException(String message) {
+        super(message);
```

Comment:
This already exists in #310

Reply:
it's not merged, not available here.

Reply:
whichever PR get's merged first will get that into master, then after refresh this change will disappear